### PR TITLE
[NOTASK]: Adding missing tests, reformatting Page not found,

### PR DESCRIPTION
**Pull request by poulami-saha**
The Contact Form unit test went missing while updating local branches.
I modified the icons back to the way it was to bring back the **alt** tag, since we are heavily prioritizing the semantic html usage.
Also updated the implementation of the Copyright text
The Page Not found was having a different background color and position of text, which has been updated to make it same like others

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -5,9 +5,9 @@ import logo from "../../assets/logo.svg";
 
 import { Link } from "react-router-dom";
 
-import { ReactComponent as FacebookIcon } from "../../assets/icon-facebook.svg";
-import { ReactComponent as PinterestIcon } from "../../assets/icon-pinterest.svg";
-import { ReactComponent as TwitterIcon } from "../../assets/icon-twitter.svg";
+import FacebookIcon from "../../assets/icon-facebook.svg";
+import PinterestIcon from "../../assets/icon-pinterest.svg";
+import TwitterIcon from "../../assets/icon-twitter.svg";
```
Comment (FevenSeyfu) [inline src/components/Footer/Footer.jsx]: Please this needs to remain the same 
```suggestion
import { ReactComponent as FacebookIcon } from "../../assets/icon-facebook.svg";
import { ReactComponent as PinterestIcon } from "../../assets/icon-pinterest.svg";
import { ReactComponent as TwitterIcon } from "../../assets/icon-twitter.svg";

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -35,13 +35,24 @@ const Footer = () => {
...
+            src={FacebookIcon}
+            className={styles.socialLink}
+            alt="facebook"
+          />
+          <img
+            src={PinterestIcon}
+            className={styles.socialLink}
+            alt="Pinterest"
+          />
+          <img src={TwitterIcon} className={styles.socialLink} alt="twitter" />
```
Comment (FevenSeyfu) [inline src/components/Footer/Footer.jsx]: This needs to remain the same to apply the hover style
```suggestion
import { ReactComponent as FacebookIcon } from "../../assets/icon-facebook.svg";
import { ReactComponent as PinterestIcon } from "../../assets/icon-pinterest.svg";
import { ReactComponent as TwitterIcon } from "../../assets/icon-twitter.svg";

**Review by FevenSeyfu — CHANGES_REQUESTED** [review]
The changes in the footer are overriding the hover style for the social links, as for alt text we can modify the component by adding aria-labels and role as image for semantics.

**Comment by poulami-saha** [comment]
> The changes in the footer are overriding the hover style for the social links, as for alt text we can modify the component by adding aria-labels and role as image for semantics.

There is a difference between aria-label and alt, details [here](https://github.com/CivicActions/accessibility/discussions/252).
 Currently we are not redirecting anywhere clicking on the social media icons, so they should not be having an hover effect. It will be considered a mistake when reviewed that nothing happens when we click here. As a image its natural that its not clickable.  Hover effect should be present only on links where there is a redirection implemented.

**Comment by FevenSeyfu** [comment]
It was added because it was included in the design. As you pointed out it comes down to semantics vs ux. so please do what you feel is best.
image

**Comment by poulami-saha** [comment]
Since its part of the design, I will keep it with hover

**Review by FevenSeyfu — APPROVED** [review]

**Pull request merged**